pkg/prowgen/configurers.go

```diff
@@ -27,7 +27,6 @@ type JobConfigurer func(*Job)
 func jobTemplate(name string, description string, configurers ...JobConfigurer) *Job {
 	job := &Job{
 		Name:     name,
-		Agent:    "kubernetes",
```

Comment:
question: This change (removing Agent) seems disconnected from the title of the PR. Could you add some detail to the PR description on why this was done?

Reply:
done